buildSrc/src/main/kotlin/ort-application-conventions.gradle.kts

```diff
@@ -164,28 +164,50 @@ tasks.named<BuildNativeImageTask>("nativeCompile") {
     }
 }
 
+val jar by tasks.getting(Jar::class)
+
+val pathingJar by tasks.registering(Jar::class) {
+    archiveClassifier = "pathing"
+
+    manifest {
+        // Work around the command line length limit on Windows when passing the classpath to Java, see
+        // https://github.com/gradle/gradle/issues/1989.
+        attributes["Class-Path"] = configurations.runtimeClasspath.get().joinToString(" ") { it.name }
```

Comment:
Can the entries have white spaces, and could this cause issues?

Reply:
We do not produce filenames with spaces, so it should not be an issue. The [spec](https://docs.oracle.com/javase/8/docs/technotes/guides/jar/jar.html#classpath) does not explicitly mention how to handle spaces in filenames, but I guess they would be URL-encoded. However, URL-encoding would encode slashes, too, which are kept literal in the examples. So given this uncertainty, I would simply go without encoding.